docs/BR.md

```diff
@@ -2691,6 +2691,7 @@ The following table details the acceptable `AttributeType`s that may appear with
 | `postalCode`             | MAY             | If present, the CA's zip or postal information. | [Section 3.2.2.1](#3221-identity) |
 | `streetAddress`          | MAY             | If present, the CA's street address. Multiple instances MAY be present. | [Section 3.2.2.1](#3221-identity) |
 | `organizationName`       | MUST            | The CA's name or DBA. The CA MAY include information in this field that differs slightly from the verified name, such as common variations or abbreviations, provided that the CA documents the difference and any abbreviations used are locally accepted abbreviations; e.g. if the official record shows "Company Name Incorporated", the CA MAY use "Company Name Inc." or "Company Name". | [Section 3.2.2.2](#3222-dbatradename) |
+| `organizationalUnitName`  | _Effective 2022-12-12_ "MUST NOT" for Root CA Certificates defined in [Section 7.1.2.1](#7121-root-ca-certificate-profile) or TLS Subordinate CA Certificates defined in [Section 7.1.2.5](#7125-technically-constrained-tls-subordinate-ca-certificate-profile) or Technically-Constrained TLS Subordinate CA Certificate defined in [Section 7.1.2.6](#7126-tls-subordinate-ca-certificate-profile). "SHOULD NOT" for all other CAs | -           | -           |
```

Comment:
Is this more clear (no worries, if it's not!)?

| `organizationalUnitName`       | -               | -           | -           |
| \ \ \ \ _Effective 2022-12-12_ for Root CA Certificates defined in [Section 7.1.2.1](#7121-root-ca-certificate-profile), TLS Subordinate CA Certificates defined in [Section 7.1.2.5](#7125-technically-constrained-tls-subordinate-ca-certificate-profile), or Technically-Constrained TLS Subordinate CA Certificates defined in [Section 7.1.2.6](#7126-tls-subordinate-ca-certificate-profile) | MUST NOT |  -  |
| \ \ \ \ _Effective 2022-12-12_ for all other CAs | SHOULD NOT | - |

We could also consider adding a third line to cover the "prior" period (i.e., "Prior to 2022-12-12").

Reply:
I'll also add that we probably should change "CAs" to "CA Certificates"

Reply:
We have a top-level effective date for this ballot of 2023-04-15, so I don't think we need another effective date here.